sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/synctask/realtime/RealtimeDataSyncTask.java

```diff
@@ -78,6 +78,7 @@ public void prepare() {
 
     @Override
     public void start(final ReportCallback callback) {
+        syncConfiguration.getReaderConfiguration().setTableNameMap(syncConfiguration.getTableNameMap());
```

Comment:
I think we can move the logic out to Controller.

Reply:
I will refactor configuration code.